lib/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "mailjet-rs"
-version = "0.1.2"
+version = "0.2.0"
```

I don't see any API breakage so far so I don't find it required to bump a minor release here.

This crate follows SemVer, unless there's any new feature as part of this set of changes I don't find why this would involve a minor release.

I suggest writing `0.1.3` instead.

```suggestion
version = "0.1.3"
```

---

Hi, my suggestion is to upgrade to version 0.2.0 because it uses hyper 0.14 instead of 0.13, which uses tokio ^1 instead of ^0 and makes a beaking change to it.
The runtime doesn't work if you are using tokio ^1 with mailjet 0.1.2.

---

Makes sense to me now. But this would be more of a breaking change then, people using `0.1.x` won't be able to use `0.2.x` if they are using Tokio at `0.x`.

I think we could keep our current Hyper RS version for Mailjet-RS at `0.x` and have Tokio `1.x` support for Mailjet-RS at `1.x` along with API changes to support Mailjet's API v3.1.

WDYT?

---

Good idea